backend/src/routes/v0/share.py

```
@@ -0,0 +1,242 @@
+)
+from src.schemas.models import ProtectedUser
+from src.services.share import ShareService
+from src.services.db import get_store, get_checkpoint_db
+from src.utils.auth import verify_credentials
+from src.utils.logger import logger
+
+router = APIRouter(tags=["Share"])
+
+
+def share_cache_key_builder(
+    func, namespace: str = "", request=None, response=None, args=(), kwargs={}
```

Ruff (0.14.11)

Unused function argument: `namespace`

(ARG001)